src/test/groovy/graphql/schema/idl/SchemaPrinterTest.groovy

```diff
@@ -1534,6 +1539,8 @@ extend input Input {
 extend input Input {
   faz: String
 }
+
+extend schema @schemaDirective
```

Comment:
When this is implemented, this line will probably get moved elsewhere. However this test is enough to show, there is no `extend schema` anywhere in the print output